ddl/dist_owner.go

```diff
@@ -222,26 +224,48 @@ func checkBackfillJobCount(sess *session, ddlJobID, currEleID int64, currEleKey
 	if err != nil {
 		return 0, errors.Trace(err)
 	}
-
-	backfillJobCnt, err = GetBackfillJobCount(sess, BackfillTable, fmt.Sprintf("ddl_job_id = %d and ele_id = %d and ele_key = %s",
-		ddlJobID, currEleID, wrapKey2String(currEleKey)), "check_backfill_job_count")
+	backfillJobCnt, err = GetBackfillJobCount(sess, BackgroundSubtaskTableSQL, fmt.Sprintf("task_key like `%d_%s_%d%%`",
+		ddlJobID, hex.EncodeToString(currEleKey), currEleID), "check_backfill_job_count")
```

Comment:
```suggestion
	backfillJobCnt, err = GetBackfillJobCount(sess, BackgroundSubtaskTableSQL, fmt.Sprintf("task_key like `%%_%s_%d_%d`",
		hex.EncodeToString(currEleKey), currEleID, ddlJobID, descStr), "check_backfill_job_count")
```

Reply:
Then it is best to wrap into a generating function, only not easy to write wrong

Reply:
Done